packages/paste-website/src/pages/components/table/index.mdx

```diff
@@ -0,0 +1,436 @@
+    <Tr>
+      <Th>First Name</Th>
+      <Th alignment="right">Last Name</Th>
+    </Tr>
+  </THead>
+  <TBody>
+    <Tr>
+      <Td>Tamir</Td>
+      <Td alignment="right">Rice</Td>
+    </Tr>
+    <Tr>
+      <Td>Philando</Td>
+      <Td alignment="right">Castile</Td>
+    </Tr>
+    <Tr>
+      <Td>Trayvon</Td>
+      <Td alignment="right">Martin</Td>
+    </Tr>
+    <Tr>
+      <Td>Breonna</Td>
+      <Td alignment="right">Taylor</Td>
+    </Tr>
+    <Tr>
+      <Td>Shantel</Td>
+      <Td alignment="right">Davis</Td>
+    </Tr>
+    <Tr>
+      <Td>Walter</Td>
+      <Td alignment="right">Scott</Td>
+    </Tr>
+  </TBody>
+</Table>`}
+</LivePreview>
+
+### Row Alignment
+
+Table rows (`Tr`) can be set with either top, middle, or bottom vertical alignment. By default they are middle-aligned.
+
+<LivePreview scope={{Table, THead, TBody, Tr, Th, Td}} language="jsx">
+  {`<Table>
+  <THead>
+    <Tr>
+      <Th width="size20">Author</Th>
+      <Th>Quote</Th>
+    </Tr>
+  </THead>
+  <TBody>
+    <Tr verticalAlignment="top">
+      <Td>Chinua Achebe</Td>
+      <Td>The impatient idealist says: ‘Give me a place to stand and I shall move the earth.’ But such a place does not exist. We all have to stand on the earth itself and go with her at her pace.</Td>
+    </Tr>
+    <Tr verticalAlignment="top">
+      <Td>James Baldwin</Td>
+      <Td>Anyone who has ever struggled with poverty knows how extremely expensive it is to be poor.</Td>
+    </Tr>
+    <Tr verticalAlignment="top">
+      <Td>Dorothy West</Td>
+      <Td>To know how much there is to know is the beginning of learning to live.</Td>
+    </Tr>
+    <Tr verticalAlignment="top">
+      <Td>W.E.B. Du Bois</Td>
+      <Td>Herein lies the tragedy of the age: not that men are poor—all men know something of poverty; not that men are wicked—who is good? Not that men are ignorant—what is truth? Nay, but that men know so little of men.</Td>
+    </Tr>
+  </TBody>
+</Table>`}
+</LivePreview>
+
+## Anatomy
+
+### Table
+
+| Property | Default token | Modifiable? |
+| -------- | ------------- | ----------- |
+| color    | color-text    | No          |
+| width    | 100%          | No          |
+
+### TBody
+
+| Property         | Default token         | Modifiable? |
+| ---------------- | --------------------- | ----------- |
+| background-color | color-background-body | No          |
+| border-width     | border-width-10       | No          |
+| border-color     | color-border-lighter  | No          |
+
+### TFoot
+
+| Property     | Default token        | Modifiable? |
+| ------------ | -------------------- | ----------- |
+| border-width | border-width-10      | No          |
+| border-color | color-border-lighter | No          |
+
+### Tr
+
+| Property             | Default token                            | Modifiable? |
+| -------------------- | ---------------------------------------- | ----------- |
+| background-color     | Even rows - color-background-row-striped | No          |
```

Comment:
Could you do "Even rows:" instead of "Even rows -"? Feels a little too dash-y haha which makes it harder to read.